Development/nmos/sdp_utils.h

```diff
@@ -482,13 +482,13 @@ namespace nmos
     {
         // additional fmtp parameters from ST 2110-21:2017
         sdp::type_parameter tp;
-        uint32_t troff; // if omitted (zero), assume default
+        bst::optional<uint32_t> troff; // if omitted (zero), assume default
```

Comment:
I assume this should have been deleted?

```suggestion
        bst::optional<uint32_t> troff; // if omitted, assume default
```